nettacker/locale/hi.yaml

```
@@ -122,4 +122,11 @@ compare_report_path_filename: तुलना रिपोर्ट सहेज
 no_scan_to_compare: तुलना करने के लिए scan_id नहीं मिला
 compare_report_saved: "{0} में तुलना परिणाम सहेजे गए"
 build_compare_report: "तुलना रिपोर्ट बनाई जा रही है"
-finish_build_report: "तुलना रिपोर्ट तैयार हो गई"
\ No newline at end of file
+finish_build_report: "तुलना रिपोर्ट तैयार हो गई"
+compare_report_path_filename: तुलना रिपोर्ट सहेजने का फ़ाइल पथ
```

YAMLlint (1.37.1)

[error] duplication of key "compare_report_path_filename" in mapping

(key-duplicates)